src/cuda/cuda_worklist/analysis.h

```diff
@@ -1,8 +1,6 @@
 #pragma once
-#include "../cuda_data.h"
 #include <set>
-
-using BitVector = int32_t;
+#include "../cuda_common.cuh"
```

Comment:
I think it best if analysis.h doesnt include cuda_common as it contains cuda specific functions, and the cuda runtime headers.